playwright-tests/tests/editor.spec.js

```diff
@@ -0,0 +1,232 @@
+      await expect(nextButton).toBeEnabled();
+
+      await description.click();
+      await description.clear();
+      await expect(nextButton).toBeDisabled();
+    });
+
+    test("should not allow invalid project account address", async ({
+      page,
+    }) => {
+      // Project Account Address
+      await page
+        .getByPlaceholder("Enter Project Account Address")
+        .fill("anyproject");
+
+      const errorMsg = await page.getByText(
+        "Invalid Near Address, please enter a valid near address",
+      );
+      await expect(errorMsg).toBeVisible();
+    });
+
+    test("cancel should navigate to main projects page", async ({ page }) => {
+      await page.getByRole("button", { name: "Cancel" }).click();
+      expect(page.url()).toContain("?page=projects");
+    });
+
+    test("should complete flow and save data correctly with images populated", async ({
+      page,
+    }) => {
+      const expectedProjectData = {
+        // TODO: We will want to get rid of this
+        title: "Sample project",
+        description: "This is a sample project",
+        profileImage: {
+          ipfs_cid: "simple_cid",
+        },
+        backgroundImage: {
+          ipfs_cid: "simple_cid",
+        },
+        tags: {
+          test: "",
+        },
+        linktree: {
+          twitter: "SampleTwitter",
+          github: "SampleGithub",
+          telegram: "SampleTelegram",
+          website: "https://www.samplewebsite.com",
+        },
+        // End remove
+        contributors: ["anybody.near", "nobody.near"],
+        tabs: ["overview", "tasks", "activity"],
+        projectAccountId: "anyproject.near",
+        teamSize: "1-10",
+        location: "anywhere",
+      };
+
+      const expectedTransactionData = {
+        "anybody.near": {
+          project: {
+            "sample-project": {
+              "": JSON.stringify(expectedProjectData),
+              metadata: {
+                name: "Sample project",
+                description: "This is a sample project",
+                image: {
+                  ipfs_cid: "simple_cid",
+                },
+                backgroundImage: {
+                  ipfs_cid: "simple_cid",
+                },
+                tags: {
+                  test: "",
+                },
+                linktree: {
+                  github: "https://github.com/SampleGithub",
+                  telegram: "https://t.me/SampleTelegram",
+                  twitter: "https://twitter.com/SampleTwitter",
+                  website: "https://www.samplewebsite.com",
+                },
+              },
+            },
+          },
+          "builddao.testnet": {
+            project: {
+              "anybody.near_project_sample-project": "",
+            },
+          },
+        },
+      };
+
+      // Page one //
+
+      // Project Account Address
+      await page
+        .getByPlaceholder("Enter Project Account Address")
+        .fill("anyproject.near");
```

Comment:
I see you already have defined an object, why don't we use the object values, it will eliminate spelling mistakes. what do you think?

Reply:
Ya I'll put this in

Reply:
I started making this change, but I don't think it's appropriate unless I create a separate object dedicated for input fields, because "expectedProjectData" is meant to be what is JSON.stringified in the project, and in a separate ticket, I've wanted to remove this duplicate data -- but first wanted to create these tests for AS-IS.

I'll see to improve this when implementing that ticket: https://github.com/NEARBuilders/gateway/issues/418